### PR TITLE
docs: apply changes from improvements in the tutorial

### PR DESCRIPTION
**Pull request by danielwiehl**

### CODE REVIEW

File: products-app/src/products-controller.ts [hunk 1]
```diff
@@ -4,11 +4,11 @@ import { Beans } from '@scion/toolkit/bean-manager';
...
 
   private products: Product[] = [
-    {id: 1, name: 'Product 1'},
-    {id: 2, name: 'Product 2'},
-    {id: 3, name: 'Product 3'},
-    {id: 4, name: 'Product 4'},
-    {id: 5, name: 'Product 5'},
+    {id: 1, name: 'Fancy Shoes'},
+    {id: 2, name: 'Shiny knick-knack'},
+    {id: 3, name: 'Facinating book'},
```
Comment (mofogasy) [inline products-app/src/products-controller.ts]: Typo: `Fascinating`

**Review by mofogasy — CHANGES_REQUESTED** [review]
Please fix typo in products list

**Review by mofogasy — APPROVED** [review]
LGTM

File: products-app/src/products-controller.ts (hunk 1 above)
Comment (danielwiehl) [reply products-app/src/products-controller.ts]: Fixed

**Pull request merged**